src/app/profile/yourGiving/stopStartRecurringGifts/step0/stopStartStep0.tpl.html

```diff
@@ -0,0 +1,80 @@
+<div class="modal-content give-modal-content">
```

Comment:
I think https://github.com/CruGlobal/give-web/blob/master/src/common/templates/giveModalWindow.tpl.html will provide this div. `<div class="modal-content give-modal-content" uib-modal-transclude></div>`